config/fxdata/objects.cfg

```diff
@@ -713,3 +713,13 @@ Properties = CHOWNED_ON_ROOM_CLAIM
 Name = SPELLBOOK_POSS
 Genre = SPELLBOOK
 Properties = CHOWNED_ON_ROOM_CLAIM
+
+[object136]
+Name = NUM_1
+Genre = DECORATION
+Properties = CHOWNED_ON_ROOM_CLAIM DESTROYED_ON_ROOM_PLACE
+ObjectSize = 200
+Image3d = "mbox.tga"
+Region3d = "0,0,58,0"
```

Comment:
How does the region work? What are the params?

Reply:
added a comment. They are x, y, width and height

Reply:
x,y as the top-left position?